### PR TITLE
Jdhrh/offering table

### PR DESCRIPTION
**Pull request by jdhrh**

### CODE REVIEW

File: src/main/java/org/candlepin/subscriptions/db/OfferingRepository.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+package org.candlepin.subscriptions.db;
+
+import org.candlepin.subscriptions.db.model.Offering;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+/**
+ * Repository interface for the Offering entity
+ */
+public interface OfferingRepository extends JpaRepository<Offering, String> {
```
Comment (kahowell) [inline src/main/java/org/candlepin/subscriptions/db/OfferingRepository.java:30]:
```suggestion
public interface OfferingRepository extends JpaRepository<Offering, String> {
    /* Intentionally left blank */
```

File: src/main/resources/liquibase/202012181510-add-offering-table.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    <changeSet id="202012181511-3" author="jharriso">
+        <comment>Create the sku to oid table</comment>
+        <createTable tableName="sku_oid">
+            <column name="sku" type="VARCHAR(255)"/>
+            <column name="oid" type="INTEGER"/>
+        </createTable>
+    </changeSet>
+
+</databaseChangeLog>
```
Comment (kahowell) [inline src/main/resources/liquibase/202012181510-add-offering-table.xml:42]: Nitpick: can has newline?

**Review by kahowell — CHANGES_REQUESTED** [review]
Overall looks good! A couple of tiny stylistic requests.

File: src/main/java/org/candlepin/subscriptions/db/OfferingRepository.java (hunk 1 above)
Comment (jdhrh) [reply src/main/java/org/candlepin/subscriptions/db/OfferingRepository.java:30]: Done!

File: src/main/resources/liquibase/202012181510-add-offering-table.xml (hunk 2 above)
Comment (jdhrh) [reply src/main/resources/liquibase/202012181510-add-offering-table.xml]: Done!

**Review by lindseyburnett — CHANGES_REQUESTED** [review]
We should add foreign keys for `sku_child_sku.sku`, `sku_child_sku.child_sku` and `sku_oid` tables back to `offering.sku`.

**Review by lindseyburnett — APPROVED** [review]

**Review by kahowell — APPROVED** [review]

**Pull request merged**